08_voting/index.tex

```diff
@@ -1 +1,101 @@
+\begin{center}
+\begin{tabular}{ |p{3cm}||p{3cm}|p{3cm}|  }
+ \hline
+ Voting Methods   & Elections of Roles & Changes of Rules   \\
+ \hline
+ Plurality   &     & \checkmark     \\
+ \hline
+ Borda Count &  \checkmark   &      \\
+ \hline
+ Runoff      &  \checkmark    &        \\
+ \hline
+ Instant Runoff    & \checkmark  &     \\
+ \hline
+ Approval  &  \checkmark    &    \\
+ 
+ \hline
+\end{tabular}    
+\end{center}
+\end{table}
+
+Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.
+
+\textbf{Plurality} will select the relative majority choices as the collective decision. In the IIGO session, the Plurality voting method will be used to decide whether a change of rule goes into effect. When the number of valid ballots (i.e. abstention excluded) is odd, the Plurality method can determine the winner without tied results. Otherwise, the solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
+
+\textbf{Borda Count} method will allow Voters to rank their preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the winner.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclude such candidate as the winner. Otherwise, the second round will proceed where the ballots will be recounted by eliminating the candidates other than the top-two. Subsequently, the candidate with the most votes between the top-two candidates will be the winner.
+
+\textbf{Instant Runoff} is a voting method which is used in the elections for a single seat which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated, and this step is repeated until only one candidate remains.
+
+\textbf{Approval} is a voting method for a single winner in which each voter selects a subset of candidates. Approval will be used for the elections of roles where each voter picks a group of candidates, and the most named or approved candidate wins.
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
```

Comment:
Suggested rewrite: Replace "there are three possible rules that can be used to" with

three possible rules can

Reply:
changed